src/App.jsx

```diff
@@ -122,20 +122,14 @@ function App() {
         )}
         {!notice.needPreventRender
           && (
-          <Switch>
-            <Route exact path={route.home}>
-              <Redirect to={route.tutorialMain} />
-            </Route>
-            <Route path={route.tutorial}>
-              <Tutorial />
-            </Route>
-            <Route exact path={route.puzzle}>
-              <Challenge />
-            </Route>
-            <Route path="*">
-              <div>{MESSAGE.NOT_FOUND}</div>
-            </Route>
-          </Switch>
+          <Routes>
```

Comment:
https://reactrouter.com/docs/en/v6/upgrading/v5#upgrade-all-switch-elements-to-routes